recipes/server_setup.rb

```diff
@@ -114,37 +121,65 @@
       end
 
       # Create option string
+      force = false
       options = ''
       case volume_values['volume_type']
+      when 'distributed'
+        Chef::Log.warn('You have specified distributed, serious data loss can occur in this mode as files are spread randomly among the bricks')
+        options = ' '
       when 'replicated'
-        # Ensure the trusted pool has the correct number of bricks available
-        if brick_count < volume_values['replica_count']
-          Chef::Log.warn("Correct number of bricks not available: #{brick_count} available, #{volume_values['replica_count']} required for volume #{volume_name}. Skipping...")
+        # Replicated can be anything from single node to X nodes. Replica_count should equal number of bricks.
+        if brick_count < 1
```

Comment:
should be `brick_count < 2`

Doing a replica of 0 or 1 causes gluster to error out